src/core/a-entity.js

```diff
@@ -211,15 +212,26 @@ var proto = Object.create(ANode.prototype, {
 
   load: {
     value: function () {
+      var self = this;
+
       if (this.hasLoaded) { return; }
+
       // Attach to parent object3D.
       this.addToParent();
+
+      // Scene load.
+      function sceneLoadCallback () { self.updateComponents(); }
       if (this.isScene) {
-        ANode.prototype.load.call(this, this.updateComponents.bind(this));
-      } else {
-        ANode.prototype.load.call(this, this.updateComponents.bind(this),
-                                  function (el) { return el.isEntity; });
+        ANode.prototype.load.call(this, sceneLoadCallback);
+        return;
+      }
+
+      // Entity load.
+      function entityLoadCallback () {
```

Comment:
any reason to? seems more consistent with doing vars-on-top since these functions get hoisted. and readability-wise, it makes sense to want to know what this function is doing in the context of the method.